lib/src/gestures/gestures.dart

```diff
@@ -566,6 +583,32 @@ abstract class MapGestureMixin extends State<FlutterMap>
     }
   }
 
+  void handleScroll(Offset position, double delta) {
+    // Makes scrolling slower when zoomed in
+    final scale =
+        3 - mapState.zoom / (mapState.options.maxZoom ?? double.infinity) * 1.5;
```

Comment:
The scroll-to-zoom ratio is to high. We need to take into account that high scroll rates could result in too rapid and unbounded zoom outside actual bounds supported by map sources. We need to take into account minZoom to ensure that both bounds are honoured if given, at that min zoom is 0 regardless of much the user zooms out. Maximum zoom should also have a practical bound to prevent the user from zooming too long inwards (zoom above 20 is not very usable, see for instance the [general zoom-level calculations made by Mapbox](https://docs.mapbox.com/help/glossary/zoom-level/))